standard/info/wikis/trackmania/info.lua

```diff
@@ -12,4 +12,6 @@ return {
 	name = 'Trackmania',
 	defaultTeamLogo = 'Trackmania_logo_lightmode.png',
 	defaultTeamLogoDark = 'Trackmania_logo_darkmode.png',
+	opponentLibrary = 'Opponent',
```

Comment:
while looking at the wiki i noticed that you also set up a `Module:Opponent/Custom`

Reply:
I discussed with its author (SyntacticSalt) and he said that he was using it before but now Opponent has the feature he implemented. So he asked me to just remove Opponent/Custom from everything and use Opponent (not removing the module yet as I didn't do PrizePool/Custom)

Reply:
hmmm
i do not see the commons one having the same functionality
the /Custom additionally fetches a players team if no team was entered and the according option is enabled

Reply:
@mbergen ^

Reply:
As the wiki does not have storage of player history that could be fetched, it is currently not needed and therefore should not be included.

Reply:
alright